### PR TITLE
Handle armv8l as armv7hl compatible userland

### PR DESCRIPTION
**Pull request by dirkmueller**
When invoking 32bit mode in userland of an aarch64 kernel the
kernel identifies itself as 'armv8l'. This is a special mode
that is minimally userland compatible with 'armv7hl' (however not
with other versions of the armvX ABI). Map this according to
how rpm(1) handles in libzypp so that zypper can be used as well.

### CODE REVIEW

**Comment by dirkmueller** [comment]
(tested via https://build.opensuse.org/project/show/home:dirkmueller:branches:openSUSE:Factory:ARM )

**Comment by ggardet** [comment]
LGTM. Thanks for the link to the tests in OBS.

**Comment by mlandres** [comment]
Thanks!

File: zypp/parser/yum/schema/common-inc.rnc [hunk 1]
```diff
@@ -52,6 +52,7 @@ private.archenum = "noarch"
             | "sparcv8"
             | "sparc"
             | "aarch64"
+            | "armv8l"
```
Comment (Vogtinator) [inline zypp/parser/yum/schema/common-inc.rnc:55]: Missing `armv8hl`

File: zypp/parser/yum/schema/common-inc.rng [hunk 2]
```diff
@@ -115,6 +115,7 @@
       <value>sparcv8</value>
       <value>sparc</value>
       <value>aarch64</value>
+      <value>armv8l</value>
```
Comment (Vogtinator) [inline zypp/parser/yum/schema/common-inc.rng:118]: Missing `armv8hl`

**Comment by Vogtinator** [comment]
The relevant RPM change is https://github.com/rpm-software-management/rpm/commit/b1f81b837f46bcbad8b527e41c7b7ebb0175d156

**Comment by mlandres** [comment]
Fixed schema files: d5e132ad3288d4f8f4cb78bf0ab41adf510bbfdf

**Pull request merged**